src/widgets/ot_color_picker.eliomi

```diff
@@ -28,6 +28,9 @@ val hsv_to_rgb : int -> float -> float -> float * float * float
 
 val make :
   ?a:[< Html_types.div_attrib > `Class ] Eliom_content.Html.attrib list ->
+  ?cp_sig:
+    (int * float * float) Eliom_shared.React.S.t
+    * (?step:React.step -> int * float * float -> unit) Eliom_shared.Value.t ->
```

Comment:
Good start ; it would probably help the user if this was also documented (see the comment below the function)

Reply:
Done !